its/ruling/src/test/resources/expected/kotlin/ktor/kotlin-S1481.json

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
This rule is a compiler diagnostic, so all FPs are due to incomplete semantic :(